spigot/src/main/java/com/github/syldium/nethertree/NetherTreePlugin.java

```diff
@@ -52,6 +53,14 @@ public DecayRunnable getRunnable(World world) {
         return this.runnablesManager.getRunnable(world);
     }
 
+    public RunnablesManager getRunnablesManager(){
+        return this.runnablesManager;
+    }
+
+    public void setRunnablesManager(RunnablesManager runnablesManager){
+        this.runnablesManager = runnablesManager;
+    }
+
```

Comment:
`RunnablesManager` is an implementation detail of the plugin. If you just want to expose a way to reload the plugin, the `JavaPlugin#reloadConfig` method can be overridden to re-read the files and re-create the appropriate classes:
```java
@Override
public void reloadConfig() {
    // Save runnables since they will be recreated
    this.runnablesManager.save();
    // Re-read the config.yml file: the default behavior of the JavaPlugin class
    super.reloadConfig();
    // Re-create our classes
    this.loadConfig();
}
```

Reply:
Attempted to do it this way in the updated version, though I wasn't sure how to do it without reorganizing the declarations a bit in the main class.